dockstore-common/pom.xml

```diff
@@ -108,11 +108,13 @@
         <!-- broad - cromwell stuff -->
         <dependency>
             <groupId>org.broadinstitute</groupId>
-            <artifactId>wdl4s-wom_2.11</artifactId>
+            <artifactId>wdl4s-wom_2.12</artifactId>
+            <version>0.15</version>
```

Comment:
You could make this and the spray version properties like you did for scala.version, but not crucial.

Reply:
I tend to use properties only when it is used by multiple artifacts, not the same artifact in multiple places (like wdl4s-wom_2.12 both here and  dockstore-webservice/pom.xml). 
In that case I prefer to put the version in `dependencyManagement` and have it be inherited from there